src/helpers/customHooks/useWeather.js

```diff
@@ -78,7 +88,23 @@ const useWeather = () => {
           }
         );
     }
-  }, [city]);
+  }, [debouncedSearchTerm]);
+
+  function fetchWeatherUsingCoordinates({ lat, lng }) {
+    fetch(
+      `https://api.openweathermap.org/data/2.5/weather?lat=${lat}&lon=${lng}&appid=${process.env.REACT_APP_APIKEY}`
+    )
+      .then((res) => res.json())
+      .then(
+        (result) => {
+          setCity(result.name);
+        },
+        (error) => {
+          setIsLoaded(true);
+          setError(error);
+        }
+      );
+  }
```

Comment:
As suggested by Priya, we are trying to use arrow functions instead of class or function keywords, to keep a cleaner and more uniform code. It will be great if you could change this function and the function LocationMarker in src/components/MyMap/index.js to an arrow function.